Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -578,10 +594,23 @@ void ABenchmarkGymGameModeBase::ReportAuthoritativePlayers_Implementation(const
 	if (HasAuthority())
 	{
 		MapAuthoritativePlayers.Emplace(WorkerID, AuthoritativePlayers);
-		ActivePlayers = 0;
+		int TotalPlayers = 0;
 		for (const auto& KeyValue : MapAuthoritativePlayers)
 		{
-			ActivePlayers += KeyValue.Value;
+			TotalPlayers += KeyValue.Value;
+		}
+		if (MapAuthoritativePlayers.Num() == NumWorkers)
+		{
+			if (AveragedTotalAuthPlayers < 0.0f)
+			{
+				AveragedTotalAuthPlayers = TotalPlayers;
+			}
+			else
+			{
+				AveragedTotalAuthPlayers = (AveragedTotalAuthPlayers + TotalPlayers) * 0.5f;
```

Comment:
SmoothedTotalAuthPlayers might make more sense here, and maybe using the formula  `SmoothedTotalAuthPlayers*(1 - x) + TotalPlayers*x` (here x = 0.5) is slightly more intuitive with what is being done

Reply:
seems good to change the x to 0.1 to make the value more smoothed and can show the real loss of clients
SmoothedTotalAuthPlayers*(1 - x) + TotalPlayers*x